pkg/services/gitopswriter/service.go

```diff
@@ -0,0 +1,207 @@
+package gitopswriter
```

Comment:
Nit: we can probably just name this file `pkg/services/gitopswriter/gitopswriter.go`, for the sole purpose of making it easier to find when fuzzy-searching in an editor.

Not a big deal either way though.

Reply:
Another one where I wasn't sure what would be better... On the one hand, your point is well taken. On the other hand, this way people browsing through the directory know where the main service logic will be found :man_shrugging: